script/Arcueid/Day 13/13_00_ARC13_3.txt

```diff
@@ -67,8 +67,7 @@ I'm struck by a splitting headache.
 [sha:4aa2a87fdb53e7fb3a840fdb97eb852434999181]{
 -- Page 238, Offset 32327.
 -- 満足に動かない体とあいまって、気分は叫び出したくなるぐらい、不快だ。
-This intense discomfort, coupled with the fact that I can't even move my body properly,
- is enough to make me want to scream.
+That together with the fact that I can't move my body properly is a discomfort almost enough to make me want to scream.
```

Comment:
I'm not sure exactly what bone was being picked with this one, "this intense discomfort" is reifying the context from the prev line, not introducing something else. Original reads cleaner to me personally.

Reply:
he may have thought the intense discomfort was a separate thought from the headache. I'm fine with either version.

That said, "...a discomfort almost enough..." is awkward and needs rephrasing if we stick with this ver.

Reply:
I agree on reverting, reads better to me.